### ISSUE TITLE
Problem: Enabling Dataverse in Storage Service disables "Add metadata files" workflow

### ISSUE THREAD

**Issue by gehurley** [archivematica/Issues#1059]
**Expected behaviour**

The workflow that enables users to add metadata files during the Ingest stage of the process typically enables them to click on the link for a particular transfer in the dashboard, browse a connected storage source, and add a metadata.csv file to the transfer for processing as [documented here](https://www.archivematica.org/en/docs/archivematica-1.10/user-manual/ingest/ingest/#uploading-metadata-csv-files-through-the-user-interface). 

**Current behaviour**

If a Dataverse transfer location is enabled in an instance via the Storage Service, and users click on the "Add metadata files" link for any transfer in the dashboard, the window shows a blank screen missing the file browser. If the transfer location is disabled in the Storage Service, the browser returns.

The Developer Tools console in Chrome shows the following error: "Uncaught SyntaxError: Invalid or unexpected token."

**Steps to reproduce**

- Run any transfer in an instance where a Dataverse space/location is enabled
- At the Ingest tab, click on the report icon on the top right of the transfer and navigate to Metadata > Add metadata files

**Your environment (version of Archivematica, operating system, other relevant details)**

Tested in 1.9.2 and 1.10.1 using Firefox and Chrome for OSX 10.14.6 and Windows 7 

Thank you!

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged (if applicable)
- [ ] Details about this issue have been added to the release notes (if applicable)

**Comment by ross-spencer** [comment]
I happened to have two other integrations easily available as transfer sources and so I was intrigued to see if:

a. The behavior here was related to transfer-type, so, say, on the AM side.
b. Happened with other storage spaces, so, say, on the storage service side. 

As it happens: 

* for different transfer types, Bags, DSpace, etc. I didn't see the same issue.
* for different storage spaces as transfer sources, I didn't see the same issue.

But for Dataverse, this was easy to recreate so I can confirm this with `1.11` as well, 🤔 strange one!! But good find Grant!

**Comment by gehurley** [comment]
Yes, definitely an odd one. Credit for noticing this goes to Emily Sommers at the University of Toronto Archives! Thanks very much for looking into it :)

**Comment by shij13** [comment]
Hello! I believe a formatting change in how the Full Path/Relative Path for Dataverse locations are input in the Storage Service fixes this issue:

When the path/query is formatted as below on two separate lines, the syntax error appears:
```
Query:*
Subtree: test
```
![Image](https://github.com/user-attachments/assets/87c8bb2b-b7ac-4953-aa91-1f2200f833fd)

But when the path/query is entered on one line as: `Query:* Subtree: test` , the syntax error goes away and the file browser appears as expected.

![Image](https://github.com/user-attachments/assets/551ec4a7-951f-43a5-b516-5adf85782fd0)

It might be useful to add a note in the documentation about this formatting requirement :)


As a related/separate issue though, the drop down menu lists the paths for the various transfer source locations rather than the human-friendly descriptions that are entered in the Storage Service, which might cause confusion for some users.
